pkg/core/hostid/gcp.go

```diff
@@ -41,7 +41,7 @@ func getMetadata(path string, cloudMetadataTimeout timeutil.Duration) string {
 
 	resp, err := c.Do(req)
 	if err != nil {
-		logrus.WithError(err).Debugf("Failed to query GCP Metadata endpoint at %s", url)
+		logrus.WithError(err).Errorf("Failed to query GCP Metadata endpoint at %s - It's possible the agent is not running on GCP", url)
```

Comment:
Should this be Info since it may trigger an error if running on AWS?

Reply:
oh yeah, that's GCP, how did I set it to Error? 🤦